padertorch/ops/losses/loss.py

```diff
@@ -105,12 +131,19 @@ def pit_loss(estimate, target, loss_fn=torch.nn.functional.mse_loss):
             f'{estimate.size()} != {target.size()}'
         )
     candidates = []
+    filler = (slice(None),) * axis 
```

Comment:
Did you mean `filler = (slice(None),) * estimate.size()`?

Reply:
No, I meant to add `axis %= estimate.ndimension()`. But there are now to possibilites to solve this. My version: `estimate[filler + (permutation, )]`, your version would be `filler[axis] = permutation` and then index with `estimate[filler]`

Reply:
Ah, you use another trick to get the axis position. Your solution is also fine.